File: c_programming/boltu_balti.c
#include<stdio.h>
int main()
{
    int n,m;
    long int sum1,sum2,t,temp;
    long long int sum;
    while(scanf("%d %d",&n,&m)==2)
    {
        if (n>m)
        {
            temp=n;
            n=m;
            m=temp;
        }
        t=n-1;
        sum1=(m*(m+1))/2;
        sum2=(t*(t+1))/2;
        sum=sum1-sum2;
        printf("Sum of %d to %d is -> %ld;\n",n,m,sum);

    }
}

File: python_programming/boltu_balti.py
try:
    while True:
        n,m=map(int,input().split())
        if n<m:
            sum1=(m*(m+1))/2
            sum2=((n-1)*n)/2
            sum=int(sum1-sum2)
            print("Sum of "+str(n)+" to "+str(m)+" is -> "+str(sum)+";")

        if n>m:
            sum1=((m-1)*m)/2
            sum2=(n*(n+1))/2
            sum=int(sum2-sum1)
            print("Sum of "+str(n)+" to "+str(m)+" is -> "+str(sum)+";")

except EOFError:
    pass
